Podfile

```diff
@@ -43,7 +43,7 @@ target 'WooCommerce' do
   pod 'CocoaLumberjack/Swift', '~> 3.5'
   pod 'XLPagerTabStrip', '~> 9.0'
   pod 'Charts', '~> 3.3.0'
-  pod 'ZendeskSDK', '~> 2.3.1'
+  pod 'ZendeskSDK', '~> 3.0'
```

Comment:
do you think we can fix the version to `3.0.1` since v3 is pretty new and we might want not always want to automatically update the version to 3.0+ whenever someone runs `pod install`?

Reply:
Will do!